### DISCUSSION TITLE
How to enable one-node mode?

### DISCUSSION THREAD

**Discussion by icefery** [longhorn/longhorn#5737] [Q&A] [answered]
> Minimum Recommended Hardware 3 nodes

Is there a way to use longhorn with just a single node.

**Comment by c3y1huang** [comment, marked as answer]
Longhorn recommends using at least three nodes in a cluster, with each volume having three replicas for redundancy. 

It is possible to have Longhorn on a single node cluster, you can achieve this by adjusting the [Default Replica Count](https://longhorn.io/docs/1.4.1/references/settings/#default-replica-count) or [Replica Node Level Soft Anti-Affinity](https://longhorn.io/docs/1.4.1/references/settings/#replica-node-level-soft-anti-affinity) base on the use cases.
